charts/seed-controlplane/charts/etcd/templates/allow-etcd-peering-network-policy.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: {{ include "networkpolicyversion" . }}
+kind: NetworkPolicy
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Ingress and Egress between etcd-{{ .Values.role }} peers.
```

Comment:
I don't understand what this policy is useful for. Can you explain?

Reply:
We don't have HA setup of etcd and it's not planned. I suggest to remove this to reduce the surface for confusion.